index.html

```diff
@@ -5,8 +5,41 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="styles.css">
 </head>
 <body>
+    <!--navigation bar-->
     
-</body>
-</html>
\ No newline at end of file
+    <div class="divNav">
+      
+        <nav class="first-Nav">
+                <img class="time" src="images/time.png" alt="time">
+                <img class="batteryGroup" src="images/battery.png" alt="battery">                 
+        </nav>      
+        <nav class="secondNav">
+                <img class="toggleIcon" src="images/toggleIcon.png" alt="toggle">                                      
+                <h3 class="logo">John Doe </h3>                                        
+        </nav>     
+    </div>
+    <!--Main body of the page-->
+    <main class="main">
+        <h1 class="mainHeader">Hey There. I'm Alex</h1>
+        <h3 class="introHeader">I'am a Software Developer</h3>
+        <p class="mainParagraph">I can help you build a product , feature or 
+            website Look through some of my work
+            and experience! If you like what you see 
+            and have a project you need coded, <br>
+            don't hestiate to contact me.</p>
+            <!--The social media icon-->
+        <div class="main-Div">
+            <img class="github" src="images\github.png" alt="github">
+            <img class="linkedIn" src="images\linkedIn.png" alt="linkedIn">
+            <img class="angelList" src="images\angelList.png" alt="angelList">
+            <img class= "twitter" src="images\twitter.png" alt="twitter">
+            <img class="medium" src="images\medium.png" alt="medium">
+        </div>
+        
+    </main>
+    <div class="bckimg">
+        <img class="sideimg"src="images/illustration-header-mobile.png" alt="illustration">
```

Comment:
[REQURED] At about 400px a white space begins to show at the right part of your app. If this happened around 900px it can be overlooked. Everything about your design is supposed to be fine up until the desktop screen size which you will work on in future.

![image](https://user-images.githubusercontent.com/62938735/198825097-b3a86486-35d9-4908-9896-248ba52e28d2.png)